Gemfile

```diff
@@ -26,3 +26,9 @@ gem 'yard'
 gem 'mysql2',  '~> 0.5'
 gem 'pg',      '~> 1.1'
 gem 'sqlite3', '~> 1.4'
+
+gem "base64", "~> 0.2.0"
+
+gem "mutex_m", "~> 0.3.0"
+
+gem "bigdecimal", "~> 3.1"
```

Comment:
This is for Ruby 3.4 but tests are failing due to how `Hash#inspect` changed so i've left fixing that for later